Makefile

```diff
@@ -1,36 +0,0 @@
-.PHONY: setup rebuild start superuser migrations migrate pull-production-data pull-production-media pull-staging-data pull-staging-media
```

Comment:
The setup command is quite important here as that sets the UID of the app container properly. This is necessary to prevent any files that Django writes from being owned by root.

Reply:
I have restored the Makefile